src/main/java/com/minecolonies/coremod/items/ItemChiefSword.java

```diff
@@ -0,0 +1,77 @@
+package com.minecolonies.coremod.items;
+
+import com.minecolonies.api.util.constant.Constants;
+import com.minecolonies.coremod.creativetab.ModCreativeTabs;
+import com.minecolonies.coremod.entity.ai.mobs.barbarians.AbstractEntityBarbarian;
+import net.minecraft.entity.Entity;
+import net.minecraft.entity.EntityLivingBase;
+import net.minecraft.entity.player.EntityPlayer;
+import net.minecraft.item.ItemStack;
+import net.minecraft.item.ItemSword;
+import net.minecraft.potion.Potion;
+import net.minecraft.potion.PotionEffect;
+import net.minecraft.world.World;
+import net.minecraftforge.fml.common.registry.GameRegistry;
+import org.jetbrains.annotations.NotNull;
+
+/**
+ * Class handling the Chief Sword item.
+ */
+public class ItemChiefSword extends ItemSword
+{
+    private static final String ITEM_NAME         = "chiefSword";
+    private static final Potion LEVITATION_EFFECT = Potion.getPotionById(25);
+    private static final Potion GLOW_EFFECT       = Potion.getPotionById(24);
+
+    private static final int GLOW_EFFECT_DURATION = 30;
+    private static final int GLOW_EFFECT_MULTIPLIER = 20;
+    private static final int GLOW_EFFECT_DISTANCE = 30;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused "GLOW_EFFECT_DISTANCE" private field. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1068)